easybuild/easyconfigs/c/CPLEX/CPLEX-12.6.3.eb

```diff
@@ -0,0 +1,18 @@
+name = 'CPLEX'
+version = '12.6.3'
+
+homepage = 'http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/'
+description = """IBM ILOG CPLEX Optimizer's mathematical programming technology enables
+ analytical decision support for improving efficiency,
+ reducing costs, and increasing profitability."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['cplex_studio%s.linux-x86-64.bin' % ''.join(version.split('.'))]
+
+dependencies = [
+    ('Java', '1.8.0_66', ''),
+    ('foss', '2015b', ''),
```

Comment:
why would you include `foss` as a dep here?

if there's a good reason, `foss` should be used as a toolchain instead?